tools/gmhazard_scripts/db_creation/empirical_db/common.py

```diff
@@ -320,19 +314,21 @@ def calculate_emp_site(
                 (
                     rupture,
                     site,
-                    rupture_df,
                     im_types,
                     tect_type_model_dict,
                     psa_periods,
                     keep_sigma_components,
                     {key: {} for key in imdb_dict.keys()},
                 )
-                for index, rupture in matching_df.iterrows()
+                for index, rupture in matching_df.iloc[:10].iterrows()
```

Comment:
Is there a reason why only the first 10 ruptures are looked at? or was this just testing?

Reply:
Good catch, that was testing